docs/extend/legacy_plugins.md

```diff
@@ -76,6 +76,7 @@ Plugin
 [Local Persist Plugin](https://github.com/CWSpear/local-persist)                    | A volume plugin that extends the default `local` driver's functionality by allowing you specify a mountpoint anywhere on the host, which enables the files to *always persist*, even if the volume is removed via `docker volume rm`.
 [NetApp Plugin](https://github.com/NetApp/netappdvp) (nDVP)                         | A volume plugin that provides direct integration with the Docker ecosystem for the NetApp storage portfolio. The nDVP package supports the provisioning and management of storage resources from the storage platform to Docker hosts, with a robust framework for adding additional platforms in the future.
 [Netshare plugin](https://github.com/ContainX/docker-volume-netshare)                 | A volume plugin that provides volume management for NFS 3/4, AWS EFS and CIFS file systems.
+[Nimble Storage Volume Plugin](https://infosight.nimblestorage.com/InfoSight/app#software/Integration+Kits/Linux+Toolkit+%28NLT%29)| A volume plug-in that integrates with Nimble Storage Unified Flash Fabric arrays. The plug-in abstracts array volume capabilities to the Docker administrator to allow self-provisioning of secure multi-tenant volumes and clones. 
```

Comment:
Per Nimble, they're working on a landing page with docs and info about the plugin. From there, they will need have a link to their login-based repo for downloads, but it the landing page will be much more user-friendly with info about the plugin. Will that be acceptable?